client.go

```diff
@@ -289,10 +310,13 @@ func (c *Client) GetFile(u string, params url.Values) (*FileResponse, error) {
 		return nil, err
 	}
 	c.addAuth(req)
+	tracer := c.newTracer()
+	tracer.Start(req)
 	re, err := c.client.Do(req)
 	if err != nil {
 		return nil, err
 	}
+	tracer.Done(&Response{code: re.StatusCode}, err)
```

Comment:
Maybe you meant `defer tracer.Done` right after `tracer.Start`? Otherwise it will not be called on the error above.